llvm-external-projects/iree-dialects/include/iree-dialects/Dialect/LinalgExt/Passes/Passes.h

```diff
@@ -246,11 +246,46 @@ std::unique_ptr<OperationPass<func::FuncOp>> createLinalgStrategyPeelPass(
         LinalgExt::LinalgTransformationFilter());
 
 /// Create a LinalgStrategyVectorizePass.
+using VectorSizeComputationFunction =
+    std::function<SmallVector<int64_t, 4>(linalg::LinalgOp, ArrayRef<int64_t>)>;
```

Comment:
nit: I recently learnt the we're following a new style, which does not need `, 4` for SmallVector. The default size is 4.